gapic/samplegen/samplegen.py

```diff
@@ -58,7 +59,28 @@
 TEMPLATE_NAME = "sample.py.j2"
 
 
-TransformedRequest = namedtuple("TransformedRequest", ["base", "body"])
+@dataclasses.dataclass(frozen=True)
+class AttributeRequestSetup:
+    value: str
+    field: Optional[str] = None
+    value_is_file: bool = False
+    input_parameter: Optional[str] = None
+    comment: Optional[str] = None
+
+
+@dataclasses.dataclass(frozen=True)
+class TransformedRequest:
+    """Class representing a single field in a method call.
+
+    The Optional[single]/Optional[body] is workaround for not having tagged unions;
```

Comment:
I read this twice and still do not understand it.

Reply:
Can you highlight what about the comment is confusing or lacking or otherwise suboptimal so I can better inform subsequent readers? I've been swimming in these waters all week, and _I_ know what I mean :stuck_out_tongue_closed_eyes: